src/dbnode/storage/namespace.go

```diff
@@ -612,8 +609,12 @@ func (n *dbNamespace) closeShards(shards []databaseShard, blockUntilClosed bool)
 
 func (n *dbNamespace) Tick(c context.Cancellable, startTime xtime.UnixNano) error {
 	// Allow the reader cache to tick.
-	n.namespaceReaderMgr.tick()
-
+	{
```

Comment:
you don't need the '{' block here, i'd put it originally just to make it clear what we were doing